.github/workflows/sourcegraph.yml

```diff
@@ -7,19 +7,20 @@ on:
 jobs:
   lsif:
     runs-on: ubuntu-latest
-    name: "Upload LSIF"
+    name: "Upload SCIP"
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-java@v3
         with:
           distribution: "temurin"
           cache: "sbt"
           java-version: 17
-      - run: |
-          curl -fL "https://github.com/coursier/launchers/raw/master/cs-x86_64-pc-linux.gz" | gzip -d > cs && chmod +x cs
-          ./cs launch com.sourcegraph:scip-java_2.13:latest.stable -M com.sourcegraph.scip_java.ScipJava -- index
+      - run: sbt dumpVersion publishLocal
+      - run: echo "addSbtPlugin(\"com.sourcegraph\" % \"sbt-sourcegraph\" % \"$(cat VERSION)\")" > project/sourcegraph_generated.sbt
```

Comment:
We reproduce the very simple steps performed by scip-java to auto-index Scala projects.